python/lsst/obs/decam/decamMapper.py

```diff
@@ -242,12 +240,12 @@ def std_raw(self, item, dataId):
         # from the metadata. Once TPV is supported, the following keyword
         # removal may not be necessary here and would probably be done at
```

Comment:
Agreed. Unfortunately the code is still required because although `makeWcs` strips those `PVn_n` keywords internally, it does so on a deep copy. I updated the comment accordingly.